### ISSUE TITLE
Cannot open Assessments from Score Page

### ISSUE THREAD

**Issue by dfuchss** [ls1intum/Artemis#6395]
### Describe the bug

Since the "Submission" Page has been removed in #6110 , we have to use the new "Score" Page to open certain assessments.

Unfortunately, this is not possible in our case:
![image](https://user-images.githubusercontent.com/22198798/228524866-048974ae-8f11-4ae0-b3f0-606cf0afa338.png)
 We get an error while clicking on open assessment (for exam tasks with two correction rounds enabled) (Error in the additional comments)

In addition, we wonder about these points:

1. If we click on open editor, we get an editor where we can modify the code and click on a submit button (even after the end of the exam)
![image](https://user-images.githubusercontent.com/22198798/228525458-b7f0fb95-2611-44d7-b14d-486f4e35c166.png)

2. How can we create the assessment of the first (or second round manually), before the merge, this was possible in the submission view. This was very useful to do some manual grading for special submissions.

### To Reproduce

See above

### Expected behavior

No error, and opening of assessment

### Which version of Artemis are you seeing the problem on?

6.1.2

### What browsers are you seeing the problem on?

Firefox

### Relevant log output

```shell
ERROR Error: Uncaught (in promise): Error: NG04002
P@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1307751
noMatchError@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1539989
apply/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1539726
D/</f<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1115041
t/this._error<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1114505
error@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1103021
_error@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1103259
error@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1103021
_error@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1103259
error@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1103021
_error@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1103259
error@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1103021
_error@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1103259
error@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1103021
_error@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1103259
error@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1103021
_error@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1103259
error@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1103021
_error@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1103259
error@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1103021
v@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1113284
_trySubscribe@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098710
subscribe/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098655
t@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1127705
subscribe@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098571
D/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1115134
t/</<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1129610
subscribe/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098619
t@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1127705
subscribe@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098571
t/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1117598
t/</<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1129610
subscribe/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098619
t@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1127705
subscribe@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098571
j@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1118298
U@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1118227
t/this._next<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1114436
next@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1102915
R/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1111824
_trySubscribe@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098710
subscribe/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098655
t@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1127705
subscribe@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098571
y@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1118497
f/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1118548
t/</<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1129610
subscribe/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098619
t@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1127705
subscribe@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098571
g/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1119228
t/</<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1129610
subscribe/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098619
t@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1127705
subscribe@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098571
Te/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1505143
t/</<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1129610
subscribe/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098619
t@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1127705
subscribe@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098571
D/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1122455
t/</<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1129610
subscribe/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098619
t@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1127705
subscribe@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098571
t/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1117598
t/</<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1129610
subscribe/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098619
t@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1127705
subscribe@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098571
t/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1117598
t/</<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1129610
subscribe/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098619
t@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1127705
subscribe@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098571
D/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1114992
t/</<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1129610
subscribe/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098619
t@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1127705
subscribe@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098571
t/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1117598
t/</<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1129610
subscribe/<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098619
t@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1127705
subscribe@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1098571
D/</<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1120986
t/this._next<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1114436
next@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1102915
D/</</m<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1121013
t/this._next<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1114436
next@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1102915
A/</<@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1111932
invoke@https://artemis.praktomat.cs.kit.edu/polyfills.6dd8c378ec2b5726.js:1:9156
onInvoke@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1412867
invoke@https://artemis.praktomat.cs.kit.edu/polyfills.6dd8c378ec2b5726.js:1:9096
run@https://artemis.praktomat.cs.kit.edu/polyfills.6dd8c378ec2b5726.js:1:4508
ue/<@https://artemis.praktomat.cs.kit.edu/polyfills.6dd8c378ec2b5726.js:1:19366
invokeTask@https://artemis.praktomat.cs.kit.edu/polyfills.6dd8c378ec2b5726.js:1:9774
onInvokeTask@https://artemis.praktomat.cs.kit.edu/main.4f1af9765d6042e7.js:1:1412680
invokeTask@https://artemis.praktomat.cs.kit.edu/polyfills.6dd8c378ec2b5726.js:1:9695
runTask@https://artemis.praktomat.cs.kit.edu/polyfills.6dd8c378ec2b5726.js:1:5131
p@https://artemis.praktomat.cs.kit.edu/polyfills.6dd8c378ec2b5726.js:1:11794
invokeTask@https://artemis.praktomat.cs.kit.edu/polyfills.6dd8c378ec2b5726.js:1:10858
O@https://artemis.praktomat.cs.kit.edu/polyfills.6dd8c378ec2b5726.js:1:23363
I@https://artemis.praktomat.cs.kit.edu/polyfills.6dd8c378ec2b5726.js:1:23661
M@https://artemis.praktomat.cs.kit.edu/polyfills.6dd8c378ec2b5726.js:1:23926
```
```

**Comment by krusche** [comment]
@JohannesStoehr can you please fix this?

**Comment by dfuchss** [comment]
Unfortunately, the issue is not fixed in version 6.1.3. I get the error 

```
ERROR Error: Uncaught (in promise): Error: NG04002
P@https://artemis.praktomat.cs.kit.edu/main.0d7d770a83b0c2cb.js:1:1304959
noMatchError@https://artemis.praktomat.cs.kit.edu/main.0d7d770a83b0c2cb.js:1:1538279
apply/<@https://artemis.praktomat.cs.kit.edu/main.0d7d770a83b0c2cb.js:1:1538011
D/</f<@https://artemis.praktomat.cs.kit.edu/main.0d7d770a83b0c2cb.js:1:1120407
t/this._error<@https://artemis.praktomat.cs.kit.edu/main.0d7d770a83b0c2cb.js:1:1119871
error@https://artemis.praktomat.cs.kit.edu/main.0d7d770a83b0c2cb.js:1:1108387
_error@https://artemis.praktomat.cs.kit.edu/main.0d7d770a83b0c2cb.js:1:1108625
error@https://artemis.praktomat.cs.kit.edu/main.0d7d770a83b0c2cb.js:1:1108387
_error@https://artemis.praktomat.cs.kit.edu/main.0d7d770a83b0c2cb.js:1:1108625
error@https://artemis.praktomat.cs.kit.edu/main.0d7d770a83b0c2cb.js:1:1108387
_error@https://artemis.praktomat.cs.kit.edu/main.0d7d770a83b0c2cb.js:1:1108625
error@https://artemis.praktomat.cs.kit.edu/main.0d7d770a83b0c2cb.js:1:1108387
```

If I try to open the first or second correction round
